jbpm-workitems/jbpm-workitems-bpmn2/src/test/java/org/jbpm/process/workitem/bpmn2/BusinessRuleTaskTest.java

```diff
@@ -167,6 +167,34 @@ public void testDecisionPassingNonNullToDMN() throws Exception {
                      output);
     }
 
+    @Test
+    public void testCallingDecisionService() throws Exception {
+        //This test make sure that null variables are passed to the DMN execution context, otherwise DMN will throw an exception that an input requirement is missing
+        KieBase kbase = readKnowledgeBase();
+        KieSession ksession = createSession(kbase);
+
+        BusinessRuleTaskHandler handler = new BusinessRuleTaskHandler(GROUP_ID,
+                                                                      ARTIFACT_ID,
+                                                                      VERSION);
```

Comment:
This `handler` is not used at all, please remove it.

Reply:
I've added a commit to correct this. Thanks